hash_practice/exercises.py

```diff
@@ -2,18 +2,40 @@
 def grouped_anagrams(strings):
     """ This method will return an array of arrays.
         Each subarray will have strings which are anagrams of each other
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: O(nlogn)? Not sure on this

```

Comment:
⏱ Time complexity will be O(n) or O(n*m), where n is the length of `strings` and m is the length of the longest element in `strings` since `sorted` is an O(m) operation. However, if we assume that all words in `strings` are valid English words, we can reduce the time complexity down to O(n) as m will be relatively small